ReSharper.FSharp/test/data/features/daemon/lambdaAnalyzer/Id.fs.gold

```diff
@@ -0,0 +1,10 @@
+﻿|fun x -> x|(0)
+|fun () -> ()|(1)
+|fun (x, y) -> (x, y)|(2)
+|fun (a, b) (c, d, e) -> (c, d, e)|(3)
+
+---------------------------------------------------------
+(0): ReSharper Warning: Lambda can be simplified
```

Comment:
What the simplified lambda is going to look like? 🙂

Reply:
@auduchinok, 
```fun (a, b) -> id```

Reply:
OK, but it doesn't look like a lambda. 🙂